Assets/Scripts/Minigames/CowardMiniGame/RipcordBehavior.cs

```diff
@@ -0,0 +1,137 @@
+/*****************************************************************************
+// File Name :         RipcordBehavior.cs
+// Author :            Mark Hanson
+// Contributors :      Marissa Moser
+// Creation Date :     5/23/2024
+//
+// Brief Description : Any function to do for the ripcord mini game can be found hear from pulling it back, pulling towards certain ranges, and counting successful pulls.
+*****************************************************************************/
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+using UnityEngine.UI;
+using PlaceboEntertainment.UI;
+
+public class RipcordBehavior : MonoBehaviour
+{
+
+    [Header("UI Stuff")]
+    [SerializeField] private TextMeshPro _successfulPulls;
+
+    [Header("Points of Movement")]
+    [SerializeField] private float _maxReach;
+    [SerializeField]private GameObject _targetFollow;
+    [SerializeField] private float _speed;
+    public bool PressedE;
```

Comment:
Does this need to be public or can it be a private serialized field?

Reply:
yeah I swap it to serialize field now